airbyte-cron/src/main/java/io/airbyte/cron/DefinitionsUpdater.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.cron;
+
+import io.airbyte.config.Configs.DeploymentMode;
+import io.airbyte.config.EnvConfigs;
+import io.airbyte.config.init.ApplyDefinitionsHelper;
+import io.airbyte.config.init.RemoteDefinitionsProvider;
+import io.airbyte.config.persistence.ConfigRepository;
+import io.micronaut.context.annotation.Value;
+import io.micronaut.scheduling.annotation.Scheduled;
+import java.net.URI;
+import javax.inject.Inject;
+import javax.inject.Singleton;
+import lombok.extern.slf4j.Slf4j;
+
+@Singleton
```

Comment:
I'm not as familiar with micronaut, is this because we want to enforce only one copy?

Reply:
I'm also not super familiar with micronaut - I just copied this from the other tasks that had been defined but yes, that was my assumption. Micronaut takes care of instantiating it once and passing the dependencies.